### PR TITLE
GH9 solved

### PR DESCRIPTION
**Pull request by aliaskov**
#1

### CODE REVIEW

File: .metadata/.plugins/org.eclipse.jdt.core/savedIndexNames.txt [hunk 1]
```diff
@@ -1,34 +1,18 @@
...
 1336396934.index
 2009188475.index
-338626639.index
 973698845.index
 144617658.index
 265370663.index
 1477847465.index
-1624768958.index
-1271342938.index
-941210201.index
```
Comment (maximorlenco) [inline .metadata/.plugins/org.eclipse.jdt.core/savedIndexNames.txt]: metadata should not be here

**Comment by aliaskov** [comment]
comment

File: .metadata/.plugins/org.eclipse.wst.sse.core/task-tags.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #
-#Wed Jul 25 20:20:17 CEST 2018
-task-tag-projects-already-scanned=2018_07_20_Person,2018_07_25_Computer,2018_07_25_stringTest2,2018_07_20_String,2018_07_25_Goats,helloWorld,2018_07_20_Book
+#Thu Jul 04 19:41:31 CEST 2019
+task-tag-projects-already-scanned=2018_08_03_Salary,docker-principals,2018_08_20_Plants,2018_07_20_String,2018_08_08_String,2018_07_25_Goats,2018_07_27_Bank,2018_07_20_Book,2018_08_03_Alchemistry,2018_08_23_Figyrki,2018_07_20_Person,2018_07_25_Computer,2018_07_25_stringTest2,2018_08_01_Animals,2018_07_27_Library,helloWorld
```
Comment (maximorlenco) [inline .metadata/.plugins/org.eclipse.wst.sse.core/task-tags.properties]: and this too

File: 04_07_2019_Temp_ampl/.classpath [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="output" path="bin"/>
+</classpath>
```
Comment (maximorlenco) [inline 04_07_2019_Temp_ampl/.classpath]: and this too

**Pull request merged**